include/fmt/os.h

```diff
@@ -129,6 +129,24 @@ class error_code {
   int get() const FMT_NOEXCEPT { return value_; }
 };
 
+// A formatter specialization for std::error_code.
```

Comment:
Let's remove this comment since it's obvious from the next line that this is a formatter specialization for `std::error_code`.